pkg/controller/plan/adapter/ovirt/client.go

```diff
@@ -56,7 +68,13 @@ func (r *Client) CreateSnapshot(vmRef ref.Ref) (snapshot string, err error) {
 			MustBuild(),
 	).Query("correlation_id", correlationID).Send()
 	if err != nil {
-		err = liberr.Wrap(err)
+		if strings.Contains(err.Error(), "Cannot create Snapshot") {
```

Comment:
posted https://github.com/oVirt/ovirt-engine-sdk-go/pull/240 
it would be useful to detect why we couldn't create the snapshot

Reply:
Added the error to the `ConflictError` so we could propagate it and print the error.

Reply:
and we can then drop the check of `strings.Contains(err.Error(), "Cannot create Snapshot")`, right?

Reply:
@bennyz another possibility is to create a temporary version of go-ovirt for 2.5 and later do it "properly" when we get https://github.com/kubev2v/forklift/pull/420 in, wdyt?

Reply:
we'll do it separately